src/main/routing/reservedTopLevelRoutePaths.ts

```diff
@@ -0,0 +1,3 @@
+const reservedTopLevelRoutePaths = ['admin', 'identity', 'user', 'vc', 'organization'];
```

Comment:
I think we should add all existing static paths like - contributors, about, profile, forum etc.

Reply:
Agreed, just makes it safe now.